ivadomed/__init__.py

```diff
@@ -1 +1 @@
-__version__ = "0.1"
+__version__ = "0.14"
```

Comment:
What are our conventions for versioning :)? Why 0.14?

Reply:
Oh I see now! The versioning goes with PyPi's release version number! I see on PyPi that the current version is PyPi 0.13 but I was only able to install 0.1. Also, I came across this error when installing, then uninstalling the package: 
![image](https://user-images.githubusercontent.com/49137243/81585309-f9d87c00-9381-11ea-9f86-47ed8765fe3c.png)

Reply:
> What are our conventions for versioning :)? Why 0.14?
No particular reason, it started from `0.1` and I had to increment a few times during my tests since pypi doesn't allow to reupload / overwrite using the same version number. 

> Also, I came across this error when installing, then uninstalling the package:
Was that in an existing venv with medicaltorch still installed or an older torchvision/pytorch? It should be fine with a new venv